### PR TITLE
Fix many typos

### PR DESCRIPTION
**Pull request by 0xced**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/Fonts/pulls) open
- [x] I have verified that I am following matches the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description

Many typos are fixed. No public classes/methods/members have any typos, only internal or private classes/methods/members are fixed meaning that no breaking change is introduced by this pull request.

### CODE REVIEW

File: tests/SixLabors.Fonts.Tests/Issues/Issues_36.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public class Issues_36
         [InlineData(3)]
         [InlineData(4)]
         [InlineData(5)]
-        public void TextWidthFroTabOnlyTextSouldBeSingleTabWidthMultipliedByTabCount(int tabCount)
+        public void TextWidthFroTabOnlyTextShouldBeSingleTabWidthMultipliedByTabCount(int tabCount)
```
Comment (dlemstra) [inline tests/SixLabors.Fonts.Tests/Issues/Issues_36.cs:18]: Thanks for all the awesome fixes. Would you be so kind to change `Fro` into `For`?

**Review by dlemstra — COMMENTED** [review]
Thanks for all these fixes! Really appreciate the effort you put into fixing these mistakes.

File: tests/SixLabors.Fonts.Tests/Issues/Issues_36.cs (hunk 1 above)
Comment (0xced) [reply tests/SixLabors.Fonts.Tests/Issues/Issues_36.cs]: I just force-pushed this fix.

**Review by JimBobSquarePants — COMMENTED** [review]
Great stuff! Really appreciate your help here!

**Review by JimBobSquarePants — APPROVED** [review]

**Pull request merged**